### ISSUE TITLE
Why SXM vs. PCI-E?

### ISSUE THREAD

**Issue by jowens** [eunaeuna/GraphFrameworkOnGPUs#7]
A reader will look at the comment that "we time only sections that are entirely on the GPU side and are part of the analytic procedure", note we're only doing single-GPU experiments, and wonder why the CPU-GPU bus matters at all. _This_ is what needs to be described in V.A.

**Comment by jowens** [comment]
The major difference in terms of Table I appears to be power consumption. Is the performance difference simply because there's 18% more power available for SXM?

**Comment by eunaeuna** [comment]
The power information is the [TBD] part. We don't have the clear answer for why the performance appears differently depending on the bus types. Please refer to the paragraph in below (part in section 6)

For each case the operating systems, device driver types, and the versions of each are identical. As mentioned in \ref{sec:experiment-setup}, performance effects as a result of CPU differences are minimal since the measurement of computation time is only conducted on the GPU side. However, any processor-specific optimizations in the driver might affect performance. Note that the CPU types are different with respect to form factor and cache size; all of these differences are encapsulated in the general interconnect types, \emph{i.e.}, PCI-E and SXM, in the rest of this section.

**Comment by jowens** [comment]
Right, so if I'm a reviewer, I jump all over that. If we don't have any answer for why these two buses give different results, we simply shouldn't put that information in the paper. Just use PCI-E.

**Comment by eunaeuna** [comment]
Actually this was the hardest part of this work. We didn't expect this SXM vs. PCI-E results... it seems Gunrock is not affected by the bus type that much comparing to Hornet. I think it is very interesting data/information to share because not many researchers have an access to all those systems like we do now....

**Comment by jowens** [comment]
It's interesting _only if_ we provide information _why_ this is happening.
